File: cpp/Bprob/under_99/100.py
d = list(map(int, input().split()))

tmp = 0
if (d[0] == 0 and d[1] >= 100):
     tmp = d[1] / 100
if (d[0] == 1 and d[1] == 100):
     tmp = 100
if (d[0] == 2 and d[1] == 100):
    tmp = 10000

print(int(pow(100, d[0]) * d[1] + tmp));

File: cpp/ABC/Bprob/under_99/100.cpp
#include <bits/stdc++.h>

using namespace std;

#define rep(i, n) for(int i = 0; i < (int)(n); i++)
#define all(x) (x).begin(), (x).end()
#define itn int
typedef long long ll;
typedef pair<int,int> p;


int main()
{
  ll d, n;
  cin >> d >> n;
  ll tmp = 0;
  if (d == 0 && n >= 100) tmp = n / 100;
  if (d == 1 && n == 100) tmp = 100;
  ll aaa = pow(100, d) * n + tmp;
  printf("%ld\n", aaa);
}
